templates/server.conf.erb

```diff
@@ -338,3 +365,46 @@ host: <%= scope['pulp::email_host'] %>
 port: <%= scope['pulp::email_port'] %>
 from: <%= scope['pulp::email_from'] %>
 enabled: <%= scope['pulp::email_enabled'] %>
+
+# = Lazy =
+#
+# Settings for lazy content loading.
+#
+# enabled:
+#   This controls whether redirect is enabled
+#
+# redirect_host:
+#   The host FQDN or IP to which requests are redirected.
+#
+# redirect_port:
+#   The TCP port to which requests are redirected
+#
+# redirect_path:
+#   The base path to which requests are redirected
+#
+# https_retrieval:
+#   boolean; controls whether Pulp uses HTTPS or HTTP to
+#   retrieve content from the streamer.
+#   WARNING: Setting this to 'false' is not safe if you wish
+#            to use Pulp to provide repository entitlement
+#            enforcement. It is strongly recommended to keep
+#            this set to 'true' and use certificates that are
+#            signed by a trusted authority on the web server
+#            that serves as the streamer reverse proxy.
+#
+# download_interval:
+#   The interval in minutes between checks for content cached
+#   by the Squid proxy.
+#
+# download_concurrency:
+#   The number of downloads to perform concurrently when
+#   downloading content from the Squid cache.
+
+[lazy]
+# enabled: false
```

Comment:
If this is set to 'enabled: false' currently, would it harm anything to uncomment this section? I think it would be better to either uncomment, or remove for now and put it into a different PR if we plan on enabling it later.